python/tvm/autotvm/database.py

```diff
@@ -122,7 +122,8 @@ def get(self, key):
     def load(self, inp, get_all=False):
         current = self.get(measure_str_key(inp))
         if current is not None:
-            current = str(current)
+            if isinstance(current, bytes):
```

Comment:
It seems better to add the check in `get` and do the conversion there, and it's handled in a single place.